src/github/client/github-client.ts

```diff
@@ -127,6 +127,14 @@ export class GitHubClient {
 				this.logger.info({ err }, "Mapping GraphQL errors to a InvalidPermission error");
 				return Promise.reject(new GithubClientInvalidPermissionsError(buildAxiosStubErrorForGraphQlErrors(response)));
 
+			} else if (graphqlErrors.find(graphqlError => graphqlError.type === "FORBIDDEN" && graphqlError.message === "has an IP allow list enabled")) {
```

Comment:
If you expand and scroll up to line 119 (I can't add a comment there), you'll see a comment where it says `// Please keep in sync with REST error mappings!!!!`

REST error mappings are being done in  https://github.com/atlassian/github-for-jira/blob/main/src/github/client/github-client-interceptors.ts#L126 - which funnily enough also has a comment saying `// Please keep in sync with GraphQL error mappings!!!!`

We were missing both SSO and BlockedIp errors from the graphql mapping.

Reply:
Tests?